<ui version="4.0" >
 <author>R. Reucher</author>
 <class>SampleChecker</class>
 <widget class="QDialog" name="SampleChecker" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Check samples</string>
  </property>
  <layout class="QGridLayout" >
   <item row="3" column="3" >
    <widget class="QCheckBox" name="checkBoxSamplesSelectGame" >
     <property name="toolTip" >
      <string>Select game in gamelist when selecting a sample set?</string>
     </property>
     <property name="text" >
      <string>Select &amp;game</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3" >
    <widget class="QPushButton" name="pushButtonHide" >
     <property name="toolTip" >
      <string>Close sample check dialog</string>
     </property>
     <property name="text" >
      <string>C&amp;lose</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QPushButton" name="pushButtonSamplesRemoveObsolete" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string>Remove obsolete sample sets</string>
     </property>
     <property name="text" >
      <string>&amp;Remove obsolete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="pushButtonSamplesCheck" >
     <property name="toolTip" >
      <string>Check samples / stop check</string>
     </property>
     <property name="text" >
      <string>&amp;Check samples</string>
     </property>
    </widget>
   </item>
   <item rowspan="5" row="1" column="2" >
    <widget class="QListWidget" name="listWidgetSamplesObsolete" />
   </item>
   <item rowspan="5" row="1" column="1" >
    <widget class="QListWidget" name="listWidgetSamplesBad" />
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="labelSamplesGood" >
     <property name="text" >
      <string>Good: 0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="labelSamplesObsolete" >
     <property name="text" >
      <string>Obsolete: 0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="labelSamplesBad" >
     <property name="text" >
      <string>Bad: 0</string>
     </property>
    </widget>
   </item>
   <item rowspan="5" row="1" column="0" >
    <widget class="QListWidget" name="listWidgetSamplesGood" />
   </item>
   <item row="4" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>105</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="9" />
 <tabstops>
  <tabstop>listWidgetSamplesGood</tabstop>
  <tabstop>listWidgetSamplesBad</tabstop>
  <tabstop>listWidgetSamplesObsolete</tabstop>
  <tabstop>pushButtonSamplesCheck</tabstop>
  <tabstop>pushButtonSamplesRemoveObsolete</tabstop>
  <tabstop>checkBoxSamplesSelectGame</tabstop>
  <tabstop>pushButtonHide</tabstop>
 </tabstops>
 <includes/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonHide</sender>
   <signal>clicked()</signal>
   <receiver>SampleChecker</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>448</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel" >
     <x>484</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
